host/ProtocolGateway.Host.Fabric/Scripts/Deploy-FabricApplication.ps1

```diff
@@ -0,0 +1,220 @@
+﻿<#
+.SYNOPSIS 
```

Comment:
We can definitely get these added, I just want to make sure all of the changes are in and accepted before changing documentation etc.